content/en/docs/concepts/storage/volumes.md

```diff
@@ -44,11 +44,11 @@ volume type used.
 
 To use a volume, specify the volumes to provide for the Pod in `.spec.volumes`
 and declare where to mount those volumes into containers in `.spec.containers[*].volumeMounts`.
-A process in a container sees a filesystem view composed from their Docker
-image and volumes. The [Docker image](https://docs.docker.com/userguide/dockerimages/)
-is at the root of the filesystem hierarchy. Volumes mount at the specified paths within
-the image. Volumes can not mount onto other volumes or have hard links to
-other volumes. Each Container in the Pod's configuration must independently specify where to
+A process in a container sees a filesystem view composed from their container
+image and volumes. The container image is at the root of the filesystem hierarchy.
+Volumes mount at the specified paths within
+the image. Volumes cannot mount onto other volumes or have hard links to
+other volumes. Each container in the Pod's configuration must independently specify where to
 mount each volume.
```

Comment:
Should we link to [`#using-subpath`](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath)?

Reply:
I'm not sure where best to link to `#using-subpath`, so please suggest otherwise if my usage doesn't make the most sense.